lms/djangoapps/grades/tests/integration/test_events.py

```diff
@@ -28,8 +28,7 @@ class GradesEventIntegrationTest(ProblemSubmissionTestMixin, SharedModuleStoreTe
     of the grading infrastructure.
     """
     @classmethod
-    def setUpClass(cls):
-        super(GradesEventIntegrationTest, cls).setUpClass()
+    def resetCourse(cls):
```

Comment:
I had to change this in order to get these tests passing routinely, as they are affected by random ordering. `test_rescoring_events` modifies the correctness of the problem, so if it is executed first, then `test_delete_student_state_events` fails because its "correct" answer gets 0 points.

To reproduce, checkout master and run `paver test_system -t lms/djangoapps/grades/tests/integration/test_events.py --disable-migrations --randomly-seed=1487193268`.

Since there are only 2 tests in this class, I now have each of them reset the course to a known state. Is that okay, or should we file a ticket to fix this more properly?